voice/bxml/verbs/playAudio.md

```diff
@@ -1,7 +1,14 @@
 {% method %}
 ## XML: `<PlayAudio>`
-The PlayAudio verb is used to play an audio file in the call.  The URL of an audio file should be included in the body
-of the `<PlayAudio>` tag.  If a relative URL is given, it is resolved relative to the endpoint that returned the BXML.
+The PlayAudio verb is used to play an audio file in the call.  The URL of an audio file should be
+included in the body of the `<PlayAudio>` tag.  If a relative URL is given, it is resolved relative
+to the endpoint that returned the BXML.
+
+Audio is cached according to [RFC 7234](https://tools.ietf.org/html/rfc7234). Our system may cache
+your media up to the value of the response's `Cache-Control` header's `max-age` directive, or,
+if none is present, until the time given in an `Expires` header. If no `Cache-Control` or `Expires`
+header is set on the response, media will not be cached. In any case, our system may always cache
+for a shorter amount of time or not cache at all.
```

Comment:
Can the last sentence be put before the previous one? It sounded like it may cache even if no `Cache-Control` or `Expires` is set (even though you just said it doesn't).